projects/plugins/boost/app/assets/src/js/features/ui/collapsible-meta/stories/collapsible-meta.stories.tsx

```diff
@@ -9,7 +9,6 @@ const meta: Meta< typeof CollapsibleMeta > = {
 		header: { control: 'text' },
 		summary: { control: 'text' },
 		editText: { control: 'text' },
-		closeEditText: { control: 'text' },
```

Comment:
The props in the storybook story are now outdated.

Run the storybook project to see what the component looks like.

To do that, go to `~projects/jetpack/projects/js-packages/storybook` (or where your monorepo project is) and run `pnpm run storybook:dev`.

After it loads, search for the `CollapsibleMeta` component and you should see it in the list. The place it's showing up is also incorrect now (maybe has been for some time, but we can update it now).

Since it's a reusable component, it shouldn't be under `Image CDN`. You can use the path in Boost which is `Plugins/Boost/Features/UI/CollapsibleMeta` instead of `Plugins/Boost/Image CDN/CollapsibleMeta` from line 6.